src/org/thoughtcrime/securesms/NicknameHandler.java

```diff
@@ -25,7 +25,7 @@ public NicknameHandler setContext(Context context) {
      * This method is implemented for only testing purposes inorder
      * to pass mocked database object
      * @param recipientDatabase
-     * @return
+     * @rezturn
```

Comment:
rezturn?

Reply:
new feature in Java X, such courage

Reply:
prolly click command z somewhere and didnt notice the change